### PR TITLE
[docs] Adds case with single store argument to sample.md

### PR DESCRIPTION
**Pull request by Tauka**
I think it's pretty important use-case, especially if we're trying to render from multiple stores, it can save us some render cycles

### CODE REVIEW

File: docs/api/effector/sample.md [hunk 1]
```diff
@@ -256,3 +256,62 @@ clickButton('click B')
...
+import {createStore, createEffect, sample, combine} from 'effector'
+
+const data = [{name: 'physics', id: 1}]
+
+const fetchContent = createEffect({
+  handler: () => new Promise(resolve => setTimeout(() => resolve(data), 0)),
+})
+
+const $lessonIndex = createStore(0)
+const $allLessons = createStore([]).on(
```
Comment (zerobias) [inline docs/api/effector/sample.md]: Tip: you can use `restore` here
https://share.effector.dev/bdAohSKM

https://github.com/zerobias/effector/blob/master/src/effector/restore.js#L34

Comment (Tauka) [reply docs/api/effector/sample.md]: Sweet, didn't know about that, should I open PR for that? This one got already merged

Comment (Tauka) [reply docs/api/effector/sample.md]: Or I guess, you can just push it to master, simple fix

Comment (zerobias) [reply docs/api/effector/sample.md]: Just a note for the future

**Pull request merged**